lib/github_advisory_sync.rb

```diff
@@ -199,110 +206,122 @@ def ghsa_id
     # so a GitHub Security Advisory ID (ghsa_id) is used instead
     def primary_id
       return cve_id if cve_id
+
       ghsa_id
     end
 
     # return a date as a string like 2019-03-21.
     def published_day
-      return nil unless github_advisory_graphql_object["publishedAt"]
+      return unless advisory["publishedAt"]
 
-      pub_date = Date.parse(github_advisory_graphql_object["publishedAt"])
+      pub_date = Date.parse(advisory["publishedAt"])
       # pub_date.strftime("%Y-%m-%d")
       pub_date
     end
 
-    def package_names
-      github_advisory_graphql_object["vulnerabilities"]["nodes"].map{|v| v["package"]["name"]}.uniq
-    end
-
-    def rubysec_filenames
-      package_names.map do |package_name|
-        File.join("gems", package_name, "#{cve_id}.yml")
-      end
-    end
-
     def withdrawn?
-      !github_advisory_graphql_object["withdrawnAt"].nil?
+      !advisory["withdrawnAt"].nil?
     end
 
     def cvss
-      return "<FILL IN IF AVAILABLE>" if github_advisory_graphql_object["cvss"]["vectorString"].nil?
-      github_advisory_graphql_object["cvss"]["score"].to_f
+      return if advisory["cvss"]["vectorString"].nil?
+
+      advisory["cvss"]["score"].to_f
     end
 
     def external_reference
-      github_advisory_graphql_object["references"].find do |ref|
+      advisory["references"].each do |ref|
```

Comment:
this is fine but i would have left `find` in place.

Reply:
also of note is iirc we often include many different references now in the data, so looking for the first non-nvd url might not always be what we want 🤔